app/modes/chat/routes.py

```
@@ -21,41 +21,44 @@ def mode(topic_name):
 
         # 1. Generate Plan if missing
-        if not topic_data or not topic_data.get('plan'):
+        if not topic_data or not topic_data.get("plan"):
             planner = PlannerAgent()
             try:
-                plan_steps = planner.generate_study_plan(
-                    topic_name, user_background)
+                plan_steps = planner.generate_study_plan(topic_name, user_background)
             except Exception:
                 # Error will be caught by global handler
                 raise
```

Ruff (0.15.6)

[warning] Remove exception handler; error is immediately re-raised

(TRY203)